### DISCUSSION TITLE
[ideas] How about adding `summary` reporter outputs at the bottom of `tar_make()` reporter messages

### DISCUSSION THREAD

**Discussion by psychelzh** [ropensci/targets#1338] [Ideas]
### Help

- [X] I understand and agree to <https://books.ropensci.org/targets/help.html>.

### Description

For now, `tar_make()` reporter prints a lot of detailed messages for the default `"verbose"` case, and the whole progress of the pipeline will be printed when `"summary"` reporter is required. Then I am wondering if we can combine these two reporters, just as when we use `apt install` (see the following screenshot). If so, we will rarely use `reporter = "summary"` because for other cases the summary information will always be printed.

![image](https://github.com/user-attachments/assets/421c26ea-0b1c-4a05-afa3-804b704977fa)

**Comment by wlandau** [comment]
I don't have plans to make reporters quite that fancy, but you can achieve pretty much the same thing by running `tar_make(reporter = "verbose")` in one process and `tar_poll()` in another.
